sqlitecluster/SQLiteNode.cpp

```diff
@@ -2278,15 +2351,21 @@ void SQLiteNode::_handlePrepareTransaction(SQLite& db, SQLitePeer* peer, const S
 
     // Are we participating in quorum?
     if (_priority) {
-        // If the ID is /ASYNC_\d+/, no need to respond, leader will ignore it anyway.
+        // If the ID is /ASYNC_\d+/, leader will keep going regardless, but we send every 10th response anyway, jsut so leader keeps relatively current with our commit count.
         string verb = success ? "APPROVE_TRANSACTION" : "DENY_TRANSACTION";
-        if (!SStartsWith(message["ID"], "ASYNC_")) {
+        uint64_t currentCommitCount = db.getCommitCount();
+        bool isAsync = SStartsWith(message["ID"], "ASYNC_");
+        bool asyncNotification = isAsync && (currentCommitCount % 10 == 0);
+        if (!isAsync || asyncNotification) {
             // Not a permafollower, approve the transaction
-            PINFO(verb << " #" << db.getCommitCount() + 1 << " (" << message["NewHash"] << ").");
+            PINFO(verb << " #" << currentCommitCount + 1 << " (" << message["NewHash"] << ").");
             SData response(verb);
-            response["NewCount"] = SToStr(db.getCommitCount() + 1);
+            response["NewCount"] = SToStr(currentCommitCount + 1);
             response["NewHash"] = success ? db.getUncommittedHash() : message["NewHash"];
             response["ID"] = message["ID"];
+            if (asyncNotification) {
+                response["AsyncNotification"] = "true";
+            }
```

Comment:
nab could do: 

```suggestion
                response["AsyncNotification"] = asyncNotification;
```

and then change your other code to test for truthiness instead of not empty.

Reply:
We could. Part of my reason for doing it this way is because it's never entirely clear if this:
```
response["AsyncNotification"] = true;
```

Ends up with the `AsyncNotification` being `"true"` or `"1"` or ASCII character `1` or what because of the way strings are handled in these maps.